e2e_tests/tests/test_sdk.py

```diff
@@ -14,20 +12,17 @@
 
 
 @pytest.mark.e2e_cpu
-def test_completed_experiment_and_checkpoint_apis() -> None:
+def test_completed_experiment_and_checkpoint_apis(create_empty_dir) -> None:
```

Comment:
The `tmp_path` and `tmp_path_factory` fixtures already exist as a part of pytest. All you need to do to use them is cite them by name.

```python
def test_completed_experiment_and_checkpoint_apis(tmp_path) -> None:
    ...
    exp = detobj.create_experiment(config, tmp_path, includes=[model_def])
```

Pretty sure that's all it needs.